### DISCUSSION TITLE
Tailscale SSH support for gokrazy

### DISCUSSION THREAD

**Discussion by bradfitz** [gokrazy/gokrazy#193] [Show and tell]
[Tailscale SSH](https://tailscale.com/tailscale-ssh/) supports Linux (most distros), macOS, and FreeBSD.

It didn't support gokrazy yet, but now does with: https://github.com/tailscale/tailscale/pull/7796

It still requires you have `github.com/gokrazy/serial-busybox` in your `config.json`.

So now you don't need to manage breakglass authorized_keys files. If the appliance is on Tailscale, you can SSH to it with Tailscale-level (wireguard identity) auth.

**Comment by damdo** [comment]
Awesome, thanks @bradfitz!
Big fan of running Tailscale on gokrazy, this will certainly come in handy 🎉
